Resources/config/sonata_admin.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="app.admin.logs" alias="Ekino\DataProtectionBundle\Admin\LogsAdmin" />
```

Comment:
Can you explain please?

Reply:
was to provide an easier code to use in sonata_admin configuration for menu for exemple but it's not absolutely necessary. removed :)